src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+
 public class Duke {
+    public String greetUser() {
+        return "Hello! I'm Duke" + "\n" + "What can I do for you?";
+    }
+
+    public String farewellUser() {
+        return "Bye. Hope to see you again soon!";
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke duke = new Duke();
+        TaskList tl = new TaskList();
+        String greeting = duke.greetUser();
```

Comment:
You may insert line breaks to improve the readability of your code, so that it is clearer what each block is doing, as opposed to a chain of random function calls lumped together.